VAMobile/src/screens/WebviewScreen/WebviewScreen.tsx

```diff
@@ -155,6 +157,14 @@ function WebviewScreen({ navigation, route }: WebviewScreenProps) {
 
         const cookieHeaders = response.headers.get('set-cookie')
         cookieHeaders && (await CookieManager.setFromResponse(AUTH_SIS_TOKEN_EXCHANGE_URL, cookieHeaders))
+
+        const cookies = await CookieManager.get(AUTH_SIS_TOKEN_EXCHANGE_URL)
+        const cookiesArray = Object.values(cookies)
+        const hasSSOCookies = SSO_COOKIE_NAMES.every((cookieName) =>
+          cookiesArray.some((cookie) => cookie.name === cookieName),
+        )
+
+        logAnalyticsEvent(Events.vama_sso_cookie_received(hasSSOCookies))
```

Comment:
Maybe we should log the screen we're going to as well?

Reply:
SSO cookie retrieval failing won't be dependent on the URL they're visiting, but it can't hurt to have that extra data. Just added that in